### PR TITLE
Add November surprise release dates

### PR DESCRIPTION
**Pull request by cwayne18**
Add in release dates for November releases

### CODE REVIEW

**Review by sftim — DISMISSED** [review]
Please avoid `N/A` as a date, as it does not localize (and we support several localizations). Try `""` instead.

File: data/releases/schedule.yaml [hunk 1]
```diff
@@ -5,12 +5,15 @@ schedules:
...
   next:
-    release: 1.28.4
+    release: 1.28.5
     cherryPickDeadline: 2023-12-01
     targetDate: 2023-12-06
   maintenanceModeStartDate: 2024-08-28
   endOfLifeDate: 2024-10-28
   previousPatches:
+    - release: 1.28.4
+      cherryPickDeadline: N/A
```
Comment (sftim) [inline data/releases/schedule.yaml:15]:
```suggestion
      cherryPickDeadline: ""
```

File: data/releases/schedule.yaml [hunk 2]
```diff
@@ -29,10 +32,13 @@ schedules:
   maintenanceModeStartDate: 2024-04-28
   endOfLifeDate: 2024-06-28
   next:
-    release: 1.27.8
+    release: 1.27.9
     cherryPickDeadline: 2023-12-01
     targetDate: 2023-12-06
   previousPatches:
+    - release: 1.27.8
+      cherryPickDeadline: N/A
```
Comment (sftim) [inline data/releases/schedule.yaml:40]:
```suggestion
      cherryPickDeadline: ""
```

File: data/releases/schedule.yaml [hunk 3]
```diff
@@ -64,10 +70,13 @@ schedules:
   maintenanceModeStartDate: 2023-12-28
   endOfLifeDate: 2024-02-28
   next:
-    release: 1.26.11
+    release: 1.26.12
     cherryPickDeadline: 2023-12-01
     targetDate: 2023-12-06
   previousPatches:
+    - release: 1.26.11
+      cherryPickDeadline: N/A
```
Comment (sftim) [inline data/releases/schedule.yaml:78]:
```suggestion
      cherryPickDeadline: ""
```

**Comment by sftim** [comment]
@cwayne I've committed to this PR as I think it helps, but you can drop that commit if you're confident it's better to have the English abbreviation.

**Review by xmudrii — CHANGES_REQUESTED** [review]
Thank you for the PR, @cwayne18! Can you please add a note for those releases to mention that those are out of band releases to fix CVE-2023-5528? We had something similar in the past, so you can search this file for `note`

**Comment by cwayne18** [comment]
Absolutely! Thanks both for the feedback, will fix ASAP

File: data/releases/schedule.yaml (hunk 3 above)
Comment (cwayne18) [reply data/releases/schedule.yaml]: Sorry, I'd seen N/A for november on the existing page so had assumed it was ok! thanks for fixing :)

**Comment by cwayne18** [comment]
Updated, thanks again!

File: data/releases/eol.yaml [hunk 4]
```diff
@@ -1,7 +1,9 @@
 branches:
   - release: "1.25"
-    finalPatchRelease: "1.25.15"
+    finalPatchRelease: "1.25.16"
     endOfLifeDate: 2023-10-28
+    note: >-
+      1.25.16 was released in November 2023 (after the EOL date) to fix [CVE-2023-5528](https://github.com/advisories/GHSA-hq6q-c2x6-hmch)
```
Comment (xmudrii) [inline data/releases/eol.yaml:6]: Let's link to our official announcement: https://groups.google.com/g/kubernetes-announce/c/c3py6Fw0DTI/m/cScFSdk1BwAJ

File: data/releases/schedule.yaml [hunk 5]
```diff
@@ -5,12 +5,17 @@ schedules:
...
     cherryPickDeadline: 2023-12-01
     targetDate: 2023-12-06
   maintenanceModeStartDate: 2024-08-28
   endOfLifeDate: 2024-10-28
   previousPatches:
+    - release: 1.28.4
+      cherryPickDeadline: ""
+      targetDate: 2023-11-14
+      note: >-
+        Out of band release to fix [CVE-2023-5528](https://github.com/advisories/GHSA-hq6q-c2x6-hmch)
```
Comment (xmudrii) [inline data/releases/schedule.yaml:18]: Let's link to our official announcement in this file too: https://groups.google.com/g/kubernetes-announce/c/c3py6Fw0DTI/m/cScFSdk1BwAJ

**Review by xmudrii — CHANGES_REQUESTED** [review]

**Comment by cwayne18** [comment]
Done, good catch thanks

**Comment by xmudrii** [comment]
Releases are out
/hold cancel

**Pull request merged**